src/main/java/use_case/autogen/PotentialTimetable.java

```diff
@@ -0,0 +1,25 @@
+package use_case.autogen;
+
+
+import entity.Section;
+import java.util.Set;
+
+public class PotentialTimetable {
+   boolean success;
+   Set<Section> timetable;
```

Comment:
im guessing you just forgot to add access modifiers since you aren't using the package-privateness anywhere i dont think

```suggestion
    private final boolean success;
    private final Set<Section> timetable;
```